ConfigREADME.md
# Config_file_profile
<p align="center"><img src="https://img.shields.io/badge/MADE%20IN BANGLADESHI-STUDENT & PROGRAMMER-green?colorA=%23ff0000&colorB=%23017e40&style=flat-square">  

    

  <h3 align="center">  

    <img src="https://emoji.discord.st/emojis/768b108d-274f-4f44-a634-8477b16efce7.gif" width="25">  

    &nbsp; WELCOME MY PROFILE &nbsp;  

    <img src="https://emoji.discord.st/emojis/768b108d-274f-4f44-a634-8477b16efce7.gif" width="25">  

  </h3>  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-1.svg">  

    

  [![Typing SVG](https://readme-typing-svg.herokuapp.com?font=Neuton&size=23&color=30FF40&background=000000&center=true&vCenter=true&width=420&height=60&lines=👋+HELLO+WORLD%2C+I'M+Sujoy+HERE+💁;😌+TODAY+I+WILL+TELL+YOU+🤗;💁+PLEASE+FOLLOW+MY+GITHUB+🙏;💻+THANKS+MY+All+FAMILY+MEMBERS+🤙+🥰)](https://git.io/typing-svg)  

    

  <img src="https://github.com/MRVIVEK-CODER/Decompiler/blob/main/106824690-8dd73a00-66ad-11eb-89e2-53e13ac6f594.gif" alt="" border="0" />  

    

  </p>  

    

  <p align="center">   

    

   𝚅𝙸𝚂𝙸𝚃𝙾𝚁𝚂 𝙲𝙾𝚄𝙽𝚃  

    

   <img src="https://profile-counter.glitch.me/Sj-Sarker/count.svg" />  

  <a href="https://api.daily.dev/get?r=Sujoy-303"><img src="https://opencollective.com/vuejs/contributors.svg?width=900" /></a>  

    

  </p>  

    

  ![](https://github-profile-summary-cards.vercel.app/api/cards/profile-details?username=Sujoy-303&theme=vue)  

    

  <img src="https://raw.githubusercontent.com/msoftware/msoftware/output/github-contribution-grid-snake.svg" />  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-1.svg">  

    

  <h3><b><i>🏆 Github Statistics :</i></b></h3>  

    

  <a href="https://github.com/Sjsarker"><img title="trophy" src="https://github-profile-trophy.vercel.app/?username=Sujoy-303&theme=monokai"></a>  

    

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐍𝐀𝐌𝐄 :- Sujoy Sarker🔥  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐀𝐆𝐄 :- 16 🔥  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐄𝐃𝐔𝐂𝐀𝐓𝐈𝐎𝐍 :- X10 🥵  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐋𝐎𝐕𝐄𝐑 :- PYTHON AND JAVA 🌞  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐑𝐄𝐋𝐈𝐆𝐈𝐎𝐍 :- Hindu 🖤  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐅𝐀𝐂𝐄𝐁𝐎𝐎𝐊 :- [Alone Sujoy ](https://www.facebook.com/sujoy.py) 🌚  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐅𝐀𝐂𝐄𝐁𝐎𝐎𝐊 𝐏𝐀𝐆𝐄 :- [Mr Devil](https://www.facebook.com/Mrdevil) 🌚  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  - 🔰 𝐆𝐈𝐓𝐇𝐔𝐁 :- [Sjsarker](https://github.com/Sjsarker) 💤  

    

  <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg">  

    

  <table border="1">  

    <tr>  

      <td valign="top"><a href="https://github.com/Sjsarker/github-readme-stats"> <img src="https://github-readme-stats.vercel.app/api?username=Sjsarker&count_private=true&show_icons=true&icon_color=FFA500&title_color=f4791f&bg_color=0,03071e,0F2027,03071e&text_color=abcdef&border_radius=10" alt ="Sujoy Sarker's GitHub stats"/></td> </a>  

      <td valign="top"> <a href="https://github.com/Sjsarker/github-readme-stats"> <img src="https://github-readme-stats.vercel.app/api/top-langs/?username=Sjsarker&layout=compact&langs_count=10" alt ="Top Languages"/></td>  

      </a>  

    </tr>  

     <tr>  

      <td colspan="2" align="center"> <a href="https://git.io/streak-stats"> <img src="http://github-readme-streak-stats.herokuapp.com?user=Sujoy-303&hide_border=true&background=f6f8fa&stroke=001427&ring=e36414&fire=e36414&currStreakNum=03045e&sideNums=03045e&currStreakLabel=03045e&sideLabels=240046&dates=fb5607&date_format=j%20M%5B%20Y%5D" alt ="Sujoy's GitHub Readme Streak Stats"/> </a>  </td>   

        

    </tr>  

  </table>  

  <br>  

    

  ## PROFILE VISITORS  

    

  <img src="https://profile-counter.glitch.me/Sjsarker/count.svg" />  

  <a href="https://api.daily.dev/get?r=Sujoy-303"><img src="https://opencollective.com/vuejs/contributors.svg?width=900" /></a>


Config.py
<p align="center"><img src="https://img.shields.io/badge/MADE%20IN BANGLADESHI-STUDENT & PROGRAMMER-green?colorA=%23ff0000&colorB=%23017e40&style=flat-square"> 
  
 <h3 align="center"> 
   <img src="https://emoji.discord.st/emojis/768b108d-274f-4f44-a634-8477b16efce7.gif" width="25"> 
   &nbsp; WELCOME MY PROFILE &nbsp; 
   <img src="https://emoji.discord.st/emojis/768b108d-274f-4f44-a634-8477b16efce7.gif" width="25"> 
 </h3> 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-1.svg"> 
  
 [![Typing SVG](https://readme-typing-svg.herokuapp.com?font=Neuton&size=23&color=30FF40&background=000000&center=true&vCenter=true&width=420&height=60&lines=👋+HELLO+WORLD%2C+I'M+Sujoy+HERE+💁;😌+TODAY+I+WILL+TELL+YOU+🤗;💁+PLEASE+FOLLOW+MY+GITHUB+🙏;💻+THANKS+MY+All+FAMILY+MEMBERS+🤙+🥰)](https://git.io/typing-svg) 
  
 <img src="https://github.com/MRVIVEK-CODER/Decompiler/blob/main/106824690-8dd73a00-66ad-11eb-89e2-53e13ac6f594.gif" alt="" border="0" /> 
  
 </p> 
  
 <p align="center">  
  
  𝚅𝙸𝚂𝙸𝚃𝙾𝚁𝚂 𝙲𝙾𝚄𝙽𝚃 
  
  <img src="https://profile-counter.glitch.me/Sj-Sarker/count.svg" /> 
 <a href="https://api.daily.dev/get?r=Sujoy-303"><img src="https://opencollective.com/vuejs/contributors.svg?width=900" /></a> 
  
 </p> 
  
 ![](https://github-profile-summary-cards.vercel.app/api/cards/profile-details?username=Sujoy-303&theme=vue) 
  
 <img src="https://raw.githubusercontent.com/msoftware/msoftware/output/github-contribution-grid-snake.svg" /> 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-1.svg"> 
  
 <h3><b><i>🏆 Github Statistics :</i></b></h3> 
  
 <a href="https://github.com/Sjsarker"><img title="trophy" src="https://github-profile-trophy.vercel.app/?username=Sujoy-303&theme=monokai"></a> 
  
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐍𝐀𝐌𝐄 :- Sujoy Sarker🔥 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐀𝐆𝐄 :- 16 🔥 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐄𝐃𝐔𝐂𝐀𝐓𝐈𝐎𝐍 :- X10 🥵 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐋𝐎𝐕𝐄𝐑 :- PYTHON AND JAVA 🌞 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐑𝐄𝐋𝐈𝐆𝐈𝐎𝐍 :- Hindu 🖤 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐅𝐀𝐂𝐄𝐁𝐎𝐎𝐊 :- [Alone Sujoy ](https://www.facebook.com/sujoy.py) 🌚 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐅𝐀𝐂𝐄𝐁𝐎𝐎𝐊 𝐏𝐀𝐆𝐄 :- [Mr Devil](https://www.facebook.com/Mrdevil) 🌚 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 - 🔰 𝐆𝐈𝐓𝐇𝐔𝐁 :- [Sjsarker](https://github.com/Sjsarker) 💤 
  
 <img align="center" alt="line" src="https://github.com/DalpatRathore/dalpatrathore/blob/main/assets/images/line-2.svg"> 
  
 <table border="1"> 
   <tr> 
     <td valign="top"><a href="https://github.com/Sjsarker/github-readme-stats"> <img src="https://github-readme-stats.vercel.app/api?username=Sjsarker&count_private=true&show_icons=true&icon_color=FFA500&title_color=f4791f&bg_color=0,03071e,0F2027,03071e&text_color=abcdef&border_radius=10" alt ="Sujoy Sarker's GitHub stats"/></td> </a> 
     <td valign="top"> <a href="https://github.com/Sjsarker/github-readme-stats"> <img src="https://github-readme-stats.vercel.app/api/top-langs/?username=Sjsarker&layout=compact&langs_count=10" alt ="Top Languages"/></td> 
     </a> 
   </tr> 
    <tr> 
     <td colspan="2" align="center"> <a href="https://git.io/streak-stats"> <img src="http://github-readme-streak-stats.herokuapp.com?user=Sujoy-303&hide_border=true&background=f6f8fa&stroke=001427&ring=e36414&fire=e36414&currStreakNum=03045e&sideNums=03045e&currStreakLabel=03045e&sideLabels=240046&dates=fb5607&date_format=j%20M%5B%20Y%5D" alt ="Sujoy's GitHub Readme Streak Stats"/> </a>  </td>  
      
   </tr> 
 </table> 
 <br> 
  
 ## PROFILE VISITORS 
  
 <img src="https://profile-counter.glitch.me/Sjsarker/count.svg" /> 
 <a href="https://api.daily.dev/get?r=Sujoy-303"><img src="https://opencollective.com/vuejs/contributors.svg?width=900" /></a>